### PR TITLE
Third Section New Sample

### PR DESCRIPTION
**Pull request by bmushtuk**
My "Testimonials" section.

### CODE REVIEW

**Comment by bmushtuk** [comment]
Please review my "Testimonials" section. There is an issue with carousel indicators

**Comment by tsukuros** [comment]
@bmushtuk please resolve conflicts so I can start my review.

File: css/main.css [hunk 1]
```diff
@@ -1,19 +1,158 @@
...
+  border: solid 1px #d4d4d4;
 
-}
\ No newline at end of file
+}
+#cta-button{
+border-radius: 3px;
+background-color: #cc5054;
+box-shadow: 0 3px 0 0 #940308;
+border: solid 1px #cc5054;
```
Comment (tsukuros) [inline css/main.css]: Indent the rules for `#cta-button`.

File: css/main.css [hunk 2]
```diff
@@ -1,19 +1,158 @@
...
-    border: solid 1px #d4d4d4;
+  padding: 35px;
+  width: 400px;
+  background-color: rgba(255, 255, 255, 0.8);
+  border: solid 1px #d4d4d4;
 
-}
\ No newline at end of file
+}
+#cta-button{
```
Comment (tsukuros) [inline css/main.css]: Use a space before an opening curly brace `{`.

File: css/main.css [hunk 3]
```diff
@@ -1,19 +1,158 @@
...
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.33;
+  letter-spacing: normal;
+  text-align: left;
+  color: #333333;
+}
+
+.bioPic {
```
Comment (tsukuros) [inline css/main.css]: Use kebab case for `CSS` names instead of camel case.
Example: change `.bioPic` -> `.bio-pic`

File: index.html [hunk 4]
```diff
@@ -13,39 +13,186 @@
...
-                    <label for="exampleInputEmail1">Download Your FREE Gantt Chart Template</label>
-                    <input type="email" class="form-control form-control-lg" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
+                    <label for="exampleInputEmail1"><strong>Download Your FREE Gantt Chart Template</strong> </label>
+                    <input type="email" class="form-control form-control-lg" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Email">
                 </div>
-                <button type="submit" class="btn btn-lg btn-danger btn-block">Submit</button>
+                <button type="submit" id="cta-button" class="btn btn-lg btn-danger btn-block">GET IT NOW!</button>
             </form>
-             
+
```
Comment (tsukuros) [inline index.html]: Remove unnecessary empty line.

**Review by tsukuros — CHANGES_REQUESTED** [review]
@bmushtuk Please fix indentation of lines and address code style issues.

File: index.html [hunk 5]
```diff
@@ -13,39 +13,186 @@
...
+    </div>
+
+
+
+
+
+
+      
+
+        <div id = "blog" class="blog">
```
Comment (tsukuros) [inline index.html]: There should be no spaces on sides of `=`.
Example: `id = "blog"` -> `id="blog"`

Comment (tsukuros) [reply index.html]: By the way, the blog section does not belong to this pull request. It should be only about the feature you are implementing.

File: css/main.css [hunk 6]
```diff
@@ -1,19 +1,158 @@
...
+  margin-right: 10%;
+  margin-top: -10%;
+  height: 90px;
+  width: 90px;
+  border-radius: 50%;
+}
+#testimonials .carousel-indicators img:active {
+  border: 2px solid #cc5054;
+  background-image: none;
+}
```
Comment (ghost) [inline css/main.css]: at the end of the code there should be an empty string.

File: index.html [hunk 7]
```diff
@@ -13,39 +13,186 @@
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+    <footer class="footer">
+        <div class="container">
+        <p>&copy;Copyright 2018 </p>
+        </div>
+    </footer>
   </body>
 </html>
```
Comment (ghost) [inline index.html]: at the end of the code there should be an empty string.

File: css/main.css [hunk 8]
```diff
@@ -1,19 +1,158 @@
...
-    background-image: url(../images/bg-hero.jpg);
-    background-size: cover;
-    min-height: 90vh;
+  background-image: url(../images/bg-hero.jpg);
+  background-size: cover;
+  min-height: 95vh;
 }
 .navbar-light .navbar-nav .nav-link {
-    color: white;
+  color: white;
```
Comment (vboiko) [inline css/main.css]: it's better to use hex values `#ffffff`

File: css/main.css [hunk 9]
```diff
@@ -1,19 +1,158 @@
...
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.33;
+  letter-spacing: normal;
+  text-align: left;
+  color: #333333;
+}
+
+.bioPic {
+  width:200px;
```
Comment (vboiko) [inline css/main.css]: indent between `:` and value

File: css/main.css [hunk 10]
```diff
@@ -1,19 +1,158 @@
...
+  height: 296px;
+}
+.h1{
```
Comment (vboiko) [inline css/main.css]: indent between class name and `{`

File: css/main.css [hunk 11]
```diff
@@ -1,19 +1,158 @@
...
+  margin-top:99px;
+}
+.skillsPic {
```
Comment (vboiko) [inline css/main.css]: don't use camel case for class names, use `.skills-pic` instead

File: css/main.css [hunk 12]
```diff
@@ -1,19 +1,158 @@
...
+  margin-right:  0px;
+}   
+li.accomplishment {
+  margin-left: 0px;   
+}
+ul {
+  list-style-type: none;
+}
+p {
+padding-bottom : 5px;
```
Comment (vboiko) [inline css/main.css]: indent before property names

File: css/main.css [hunk 13]
```diff
@@ -1,19 +1,158 @@
...
+.carousel-slide {
+  max-height: 415px;
+}
+.carousel-item::before {
+  content: url(../images/quotes.svg);
+  width: 45px;
+  height: 36px;
+  position: relative;
+  z-index: 1;
+  padding-left: 17%;
```
Comment (vboiko) [inline css/main.css]: don't use `%`  for padding values

**Review by ghost — COMMENTED** [review]
at the end of the code there should be an empty string.

File: index.html [hunk 14]
```diff
@@ -13,39 +13,186 @@
...
+            </div>
+        </div>
+    </div>
+
+
+
+
+
+
+
```
Comment (vboiko) [inline index.html]: what's the spacing for?

File: index.html (hunk 7 above)
Comment (vboiko) [inline index.html]: add a line

File: css/main.css [hunk 15]
```diff
@@ -1,19 +1,150 @@
...
+  padding: 35px;
+  width: 400px;
+  background-color: rgba(255, 255, 255, 0.8);
+  border: solid 1px #d4d4d4;
+}
+#cta-button {
+border-radius: 3px;
+background-color: #cc5054;
+box-shadow: 0 3px 0 0 #940308;
+border: solid 1px #cc5054;
```
Comment (tsukuros) [inline css/main.css]: Indent lines inside a CSS selector declaration. Example:
```
#cta-button {
  border-radius: 3px;
  background-color: #cc5054;
  box-shadow: 0 3px 0 0 #940308;
  border: solid 1px #cc5054;
}
```

File: css/main.css (hunk 6 above)
Comment (tsukuros) [reply css/main.css]: @melnichenkod This is a very good point. VS code can do it automatically if you set it User Settings. All you need to do is add this one line:
Steps:
1. Press CMD(CTRL) + ','
2. Paste the following line
```
"files.insertFinalNewline": true
```
Here is my example: http://prntscr.com/iom5cd

File: css/main.css [hunk 16]
```diff
@@ -1,19 +1,150 @@
...
+padding-bottom: 5px;
+}
+.carousel-content {
+  color: #000000;
+  display: flex;
+}
+#text-carousel {
+width: 80%;
+position: relative;
+padding: 70px;
```
Comment (tsukuros) [inline css/main.css]: Indent rules inside of CSS declaration.

File: css/main.css [hunk 17]
```diff
@@ -1,19 +1,150 @@
...
+}
+.testimonials {
+background-color: #f5f5f5;
```
Comment (tsukuros) [inline css/main.css]: Indent rules inside of CSS declaration.

File: css/main.css [hunk 18]
```diff
@@ -1,19 +1,150 @@
...
+  margin-right:  0px;
+}   
+li.accomplishment {
+  margin-left: 0px;   
+}
+ul {
+  list-style-type: none;
+}
+p {
+padding-bottom: 5px;
```
Comment (tsukuros) [inline css/main.css]: Indent rules inside of CSS declaration.

File: css/main.css (hunk 6 above)
Comment (bmushtuk) [reply css/main.css]: Ok, thank you!

Comment (bmushtuk) [reply css/main.css]: However, it doesn't seem to be recognized while making a commit (even though it is now automatically created by VS Code itself)

Comment (bmushtuk) [reply css/main.css]: empty line

**Review by ghost — APPROVED** [review]

**Review by tsukuros — APPROVED** [review]

File: index.html [hunk 19]
```diff
@@ -88,61 +88,128 @@ <h1 class="h1 text-center">About Me</h1>
...
+                            <img class="img-fluid" src="images/arrow-left.svg" alt="Arrow Right Icon" />
+                        </span>
+                        <span class="sr-only">Next</span>
+                    </a>
+                </div>
             </div>
         </div>
     </section>
+
+    <section id="resources" class="pt-5 pb-5">
```
Comment (tsukuros) [inline index.html:147]: @bmushtuk This section shouldn't be on this branch. Instead, you should have made the following steps:
1) Merge `test-third-section` branch into `master`.
2) Create a new branch off of `master`.
3) Create a Resources section and others
4) Try to make smaller changes per commit for easier review. Like a commit per section.

Now you need to google how to move this commit to a new branch.

**Pull request merged**